cmd/appliance/appliance.go

```diff
@@ -27,6 +28,8 @@ func NewApplianceCmd(f *factory.Factory) *cobra.Command {
 	pFlags := cmd.PersistentFlags()
 	pFlags.StringToStringP("include", "i", map[string]string{}, "Include appliances. Adheres to the same syntax and key-value pairs as '--exclude'")
 	pFlags.StringToStringP("exclude", "e", map[string]string{}, filterHelp)
+	pFlags.StringSlice("order-by", []string{"name"}, orderByHelp)
+	pFlags.Bool("descending", false, "Change the direction of sort order when using the '--order-by' flag. Using this will reverse the sort order for all keywords specified in the ''--order-by' flag.")
```

Comment:
these flags are not applicable for all the appliance commands

Reply:
Could that code be reused for the relevant commands instead?
E.g. some kind of "arg group" that you can pass to the appropriate subcommands.

Reply:
Not that I know of, the easiest way would be to hide from the commands where it's not applicable. On the other hand, there are very few appliance commands that doesn't list appliances at all. I'm not sure I understand really which appliance commands this wouldn't be applicable to?

Reply:
wouldn't it be easier to just have these flags in stats and appliance list command where there are used?

Reply:
There are lists being used in basically every appliance command. If it doesn't output a list, it will prompt with a list if you don't use arguments, or output the list in a summary. Even when prompting, being able to order by specific values is useful.

Reply:
Do you mean even including inside the prepare upgrade etc.. this will sort appliances according to the flag?

Reply:
yes, that's what I'm saying. Complete has it's own logic for grouping the appliances, but the flags will affect how the appliances are listed within the groups